wireshark/USAGE.md

```diff
@@ -0,0 +1,39 @@
+# OQS-wireshark
+
+This docker image contains a version of [wireshark](https://www.wireshark.org/) built to also properly display quantum-safe crypto (QSC) TLS operations.
+
+To this end, it contains references to algorithms supported by [liboqs](https://github.com/open-quantum-safe/liboqs) and [OQS-OpenSSL](https://github.com/open-quantum-safe/openssl) from the [OpenQuantumSafe](https://openquantumsafe.org) project.
+
+The image is based on Ubuntu and requires the host to run the Unix X-Window system.
+
+## Quick start
+
+Execute this command to open the wireshark window on your host:
+
+    docker run --net=host --privileged --env="DISPLAY" --volume="$HOME/.Xauthority:/root/.Xauthority:rw" openquantumsafe/wireshark
+
+*Note:* Select a network interface to monitor.
+
+## Suggested test
+
+At https://test.openquantumsafe.org most quantum-safe algorithms that are still part of the NIST PQC competition are available for TLS interoperability testing.
+
+As a client, we recommend using an OQS-enabled `curl` docker image that may be executed for a quick initial test as follows:
+
+     docker run -it openquantumsafe/curl sh -c "curl -k https://test.openquantumsafe.org:6120 --curves p521_frodo1344aes"
+
+For more details regarding the client-side options, we recommend reviewing https://hub.docker.com/repository/docker/openquantumsafe/curl
+
+### Focus on quantum-safe data traffic 
+
+In order to focus `wireshark` on the quantum safe TLS traffic generated by the above `curl` command, we recommend setting a wireshark display filter as follows
+
+     ip.addr == 149.81.106.123
```

Comment:
I suggest filtering by TLS to make it easier to see that traffic:

```suggestion
     ip.addr == 149.81.106.123 && tls
```

Reply:
Good improvement. Thx.